### PR TITLE
Releasing.md instructions to uncomment nexus-staging-maven-plugin

### PR DESCRIPTION
**Pull request by andreamlin**

### CODE REVIEW

File: RELEASING.md [hunk 1]
```diff
@@ -102,7 +102,7 @@ This script checks out `gh-pages` branch of the repository, builds the documenta
 
 11. Check that you are not trying to release a SNAPSHOT build (the artifacts versions do not have "-SNAPSHOT" suffix) and then run `mvn clean deploy -DskipTests=true --settings ~/.m2/settings.xml -P release` command. It will build and deploy artifacts to the staging repository. 
 
-12. Run `mvn nexus-staging:release` to release the artifacts. 
+12. Uncomment the `nexus-staging-maven-plugin` plugin definition from step 10.2; This plugin needed to release the artifacts. Run `mvn nexus-staging:release` to release the artifacts.
```
Comment (garrettjonesgoogle) [inline RELEASING.md]: s/This plugin/This plugin is/

Comment (andreamlin) [reply RELEASING.md]: Done.

**Comment by andreamlin** [comment]
Fixed typo. PTAL

**Comment by andreamlin** [comment]
Ok, fixed the grammar. PTAL

**Pull request merged**